include/net/socket.h

```diff
@@ -207,6 +207,16 @@ static inline void zsock_freeaddrinfo(struct zsock_addrinfo *ai)
 	free(ai);
 }
 
+#define NI_NUMERICHOST 1
```

Comment:
Wouldn't there be a namespace conflict if this value comes from newlib for example?

Reply:
> Wouldn't there be a namespace conflict if this value comes from newlib for example?

So, newlib doesn't have any socket implementation (nor related POSIX headers).

But you caught me - while I was the advocate of the socket subsys namespacing, I get a bit weary of all this renaming. Partly because it will need to be done twice - for CONFIG_NET_SOCKETS_POSIX_NAMES the adhoc option, and then for CONFIG_POSIX_API (and in proper POSIX headers, I'm working on that).

So, I'm still strict on function namespacing, because otherwise we'll have mess, but I'm becoming more lax on namespacing of aux stuff like these (pseudo)enums.

So, that's full disclosure, feel free either insist on namspacing these consistently, or feel free to pass by this ;-).

Reply:
@jukkar: So, what do you say? Given that merges continue so far, and this is pretty standalone changes, would be nice to get that into 1.14.

Reply:
Let's deal with the namespace issue if we encounter that.